lib/predictions/store/behaviour.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Predictions.Store.Behaviour do
```

Comment:
question: could we come up with a different name for this behaviour? I find it confusing to read code with `@behaviour Behaviour` everywhere, it makes it seem like this is some kind of generic library code for behaviours but it seems like it's actually a Predictions Store Behaviour.

Reply:
I actually think it is pretty straightforward. `Predictions.Store` implements the `Predictions.Store.Behaviour`. Makes it so that any time I want to find the custom behaviour being implemented, I know right where to look.